### PR TITLE
Deduplicate shared logic in `TimelineController`

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: app/controllers/timeline_controller.rb [hunk 1]
```diff
@@ -4,36 +4,30 @@ class TimelineController < ApplicationController
...
-      format.html
-      format.js { render layout: false }
-    end
+    @title = list_title(current_user.lists.find_by!(name: params[:list_name]))
+    paginate_timeline { |args| @list.cursored_timeline(**args) }
   end
 
   def public
-    @timeline = Answer.cursored_public_timeline(last_id: params[:last_id])
+    @title = generate_title("Public Timeline")
```
Comment (pixeldesu) [inline app/controllers/timeline_controller.rb]: Missing locale!

**Review by pixeldesu — CHANGES_REQUESTED** [review]

File: app/controllers/timeline_controller.rb (hunk 1 above)
Comment (raccube) [reply app/controllers/timeline_controller.rb]: Good catch

**Review by pixeldesu — APPROVED** [review]

**Review by pixeldesu — APPROVED** [review]

File: app/views/timeline/timeline.haml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-= render 'shared/cursored_pagination_dummy', more_data_available: @more_data_available, last_id: @timeline_last_id
+= render "shared/cursored_pagination_dummy", more_data_available: @more_data_available, last_id: @timeline_last_id
 
 - if @more_data_available
   .d-flex.justify-content-center.justify-content-sm-start
     %button.btn.btn-light#load-more-btn{ type: :button, data: { last_id: @timeline_last_id } }
       = t 'views.actions.load'
 
-- provide(:title, APP_CONFIG['site_name'])
+- provide(:title, @title ||= APP_CONFIG["site_name"])
```
Comment (nilsding) [inline app/views/timeline/timeline.haml]: what's that `@title ||=` for?

**Review by nilsding — APPROVED** [review]
Lemons Greet The Monarch

**Pull request merged**